zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -4331,7 +4331,11 @@ components:
             $like:
               description: |
                 Checks if the property matches the provided like value.
-                Supported wildcard characters depend on the configured search client.
+
+                Supported wildcard characters are:
+
+                * `*`: matches zero, one, or multiple characters.
+                * `?`: matches one, single character.
```

Comment:
👍 So much better to be able to claim this

Reply:
Should we explain how to search for values that contain those values already, i.e. how to escape this in search queries in the API so I can find `my*value` using like search?